libs/elektra/Makefile

```diff
@@ -149,7 +146,7 @@ endef
 define Package/libelektra-crypto
   $(call Package/libelektra/Default)
   TITLE:=Elektra crypto plugin
-  DEPENDS:=+libelektra-core +libopenssl
+  DEPENDS:=+libelektra-core +libgcrypt
```

Comment:
Curious why.

Reply:
It's an upstream decision. They don't support libopenssl anymore. Maybe @markus2330 can fill in the details?

It seems CI fails after 45 minutes, when it is still building boost. Just runs out of time?

I hope this is not a problem for merging the PR.

Reply:
No it’s a RAM problem. I ran into the same issue locally with my VM. I’ll merge later today.

Reply:
We needed to put too much effort into maintenance, so we only kept the best plugin as evaluated by the author (@petermax2): https://www.libelektra.org/ftp/elektra/publications/nirschl2018cryptographic.pdf